<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_departement</class>
 <widget class="QDialog" name="Dialog_departement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>521</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(44, 62, 89);
border-right-color: rgb(44, 62, 89);
border-color: rgb(44, 62, 89);
selection-background-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-bottom-color: rgb(44, 62, 89);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab_Ajouter">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_L">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Ide">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>132</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);

</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Cd">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>68</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="Prenom">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>132</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Identifiant categorie</string>
     </property>
    </widget>
    <widget class="QLabel" name="ide_auteur">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>68</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Code Departement</string>
     </property>
    </widget>
    <widget class="QLabel" name="nom_auteur">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>100</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Libelle Departement</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Supprimer">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>58</y>
       <width>410</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Vous devez Tapez l'identifiant de departement que vous souhaite le supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ide_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>valider</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Modifer">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>60</y>
       <width>400</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Vous devez Tapez l'identifiant de departement que vous souhaite modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ide_modif">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>80</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_L_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>148</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QLabel" name="Prenom_5">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>180</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Identifiant categorie</string>
     </property>
    </widget>
    <widget class="QLabel" name="nom_auteur_5">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>148</y>
       <width>120</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Libelle Departement</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Ide_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);

</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Consulter">
    <attribute name="title">
     <string>Consulter</string>
    </attribute>
    <widget class="QTableView" name="tabledepartement">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>440</width>
       <height>210</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Trier">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_chercher_1">
    <attribute name="title">
     <string>Chercher Par saisi </string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_identifiant">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_saisi">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>250</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Vous devez Tapez l'identifiant de departement</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabledepartement_chercher">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>460</width>
       <height>210</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chercher">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>290</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
